docs/standard/serialization/system-text-json/snippets/how-to-contd/vb/Program.vb

```diff
@@ -14,49 +14,49 @@
             ImmutableTypes.Program.Main()
             Console.WriteLine()
 
-            Console.WriteLine("======== Field support =========")
-            Fields.Program.Main()
-            Console.WriteLine()
+            'Console.WriteLine("======== Field support =========")
```

Comment:
Why is this being commented out?

Reply:
This code isn't shown in the docs anywhere. It's just a way to run the different snippets/methods. I didn't want to have to run all of them - just the one I was adding.